### ISSUE TITLE
Floating panels are topmost

### ISSUE THREAD

**Issue by KarlSevenSeven** [twinbasic/twinbasic#1298]
**Describe the bug**
Floating panels are topmost even if tB doesn't have the focus.

**To Reproduce**
1. Click Window - Create ToolWindow - Outline
2. Set the focus to another program

**Expected behavior**
Floating panel to behave like the main tB window

**Screenshots**
![image](https://user-images.githubusercontent.com/88059532/197739351-38cbb7bd-60f5-46aa-a559-420c50bfc0c9.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - twinBASIC 157

**Comment by WaynePhillipsEA** [comment]
Yes, this is a known issue.  With the tB IDE being built with tB itself, we can't yet support non-modal forms, so instead the toolwindows all run in separate threads.  In this configuration, the toolwindows can't be parented by the main IDE form, and so making them all topmost was the simplest solution to keep them visible for now.

Once tB supports non-modal forms, we can change this.

**Comment by wqweto** [comment]
Does setting owner with `Call SetWindowLong(hToolWindow, GWL_HWNDPARENT, hMainForm)` between windows on different threads fail?

This might allow `hToolWindow` to always stay in front of `hMainForm` without setting the thermo-nuclear option of repositioning in front of `HWND_TOPMOST`.

**Comment by KarlSevenSeven** [comment]
>  thermo-nuclear option
ROFL

**Comment by WaynePhillipsEA** [comment]
> Does setting owner with `Call SetWindowLong(hToolWindow, GWL_HWNDPARENT, hMainForm)` between windows on different threads fail?
> 
> This might allow `hToolWindow` to always stay in front of `hMainForm` without setting the thermo-nuclear option of repositioning in front of `HWND_TOPMOST`.

Unfortunately calling SetParent (or GWL_HWNDPARENT) just causes both UI threads to hang.
